client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -365,6 +366,10 @@ export class Checkout extends React.Component {
 			return `/checklist/${ selectedSiteSlug }`;
 		}
 
+		if ( this.props.isJetpack && isEnabled( 'jetpack/checklist' ) ) {
```

Comment:
We should be more careful with Atomic:

https://github.com/Automattic/wp-calypso/blob/6104bb69c5bdc96093bfa6339182f1557b2652c0/client/my-sites/checkout/checkout/checkout.jsx#L363